tools/python/boutiques/localExec.py

```diff
@@ -15,6 +15,7 @@
 from termcolor import colored
 from boutiques.evaluate import evaluateEngine
 from boutiques.logger import raise_error, print_info
+from boutiques.dataHandler import getDataCacheDir
```

Comment:
This line is currently breaking pytest's setup. There is an importError here, I am still troubleshooting the issue.

Reply:
Hey @friesenA - I believe this is due to not adding the module to the [`__init__.py`](https://github.com/friesenA/boutiques/blob/develop/tools/python/boutiques/__init__.py) file, so the import is having trouble. The reason this may not be popping up in normal use is because the relative location of the modules is the same directory, so they "get lucky" and import it locally.

Reply:
Thanks, this issue has now been resolved.